cypress/integration/06.Search.feature

```diff
@@ -8,7 +8,7 @@ Feature: Search
     And we have the following posts in our database:
       | Author        | id | title                                         | content                                 |
       | Brianna Wiest | p1 | 101 Essays that will change the way you think | 101 Essays, of course!                  |
-      | Brianna Wiest | p1 | No searched for content                       | will be found in this post, I guarantee |
+      | Brianna Wiest | p2 | No searched for content                       | will be found in this post, I guarantee |
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-03-26T22:59:06Z" title="Tuesday, March 26th 2019, 11:59:06 pm +01:00">Mar 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -8,7 +8,7 @@ Feature: Search
     And we have the following posts in our database:
       | Author        | id | title                                         | content                                 |
       | Brianna Wiest | p1 | 101 Essays that will change the way you think | 101 Essays, of course!                  |
-      | Brianna Wiest | p1 | No searched for content                       | will be found in this post, I guarantee |
+      | Brianna Wiest | p2 | No searched for content                       | will be found in this post, I guarantee |
```


thanks for fixing this, I must have been crazy to create two posts with the same id!!